### ISSUE TITLE
[Android]Null exception throw from ScrollViewRenderer.Dispose(bool disposing) in Xamarin.Forms Android

### ISSUE THREAD

**Issue by jessejiang0214** [xamarin/Xamarin.Forms#2011]
### Steps to Reproduce

1. Put Scrollview into ListView header
2. Set Scrollview.Orientation="Horizontal" and add some controls into the scrollview
3. pop the page

### Expected Behavior
No exception
### Actual Behavior
Throw null exception, If I catch the null exception from override  Dispose function, it will throw other exception from RemoveAllViews. I found the source code of ScrollViewRenderer 
```
		protected override void Dispose(bool disposing)
		{
			base.Dispose(disposing);

			SetElement(null);

			if (disposing)
			{
				Tracker.Dispose();
				Tracker = null;
				RemoveAllViews();
				_container.Dispose();
				_container = null;
			}
		}
```
It didn't check if Tracker is null.
### Basic Information

- Version with issue: 2.5.0.121934
- Last known good version:
- IDE: Visual Studio for Mac 7.3.3
- Platform Target Frameworks: 

Xamarin.Android
Version: 8.1.5.0 (Visual Studio Community)
Android SDK: /Users/jesse/Library/Developer/Xamarin/android-sdk-macosx
	Supported Android versions:
		6.0 (API level 23)
		7.0 (API level 24)
		7.1 (API level 25)
		8.0 (API level 26)

SDK Tools Version: 26.1.1
SDK Platform Tools Version: 27.0.1
SDK Build Tools Version: 26.0.3

- Nuget Packages: N/A
- Affected Devices: All

**Comment by hartez** [comment]
[Repro](https://github.com/xamarin/Xamarin.Forms/files/1783398/GitHub2011.Repro.zip)

**Comment by PureWeen** [comment]
Bugzilla32462 in the control gallery

**Comment by rmarinho** [comment]
ok I got it we have a fix #3025
